src/utils/rocksdb_serialization.hpp

```diff
@@ -182,9 +182,13 @@ inline std::string SerializeVertexAsValueForLabelIndex(storage::LabelId indexing
   return SerializeVertexAsValueForAuxiliaryStorages(indexing_label, vertex_labels, property_store);
 }
 
-inline std::vector<storage::LabelId> DeserializeLabelsFromLabelIndexStorage(const std::string &value) {
-  const auto value_splitted = value.substr(0, value.find('|'));
-  return TransformFromStringLabels(utils::Split(value_splitted, ","));
+inline storage::LabelId DeserializeLabelFromIndexStorage(const std::string &value) {
+  std::string label = value.substr(0, value.find('|'));
+  return storage::LabelId::FromUint(std::stoull(label));
```

Comment:
Are we sure that in value we have just one label?

Reply:
yes, you can check serialization methods for LabelIndex and LabelPropertyIndex

Reply:
ah Monday bloody Monday... You are right this is for deserializing labels from value string

Reply:
and then there was the bug that when we deserialized labels, we didn't include indexing label :exploding_head: